packages/realm-server/migrations/1728561353454_create-user-billing.js

```diff
@@ -0,0 +1,164 @@
+exports.up = (pgm) => {
+  pgm.createTable('users', {
```

Comment:
Next related task would be to create this record when someone registers, and backfill users that we can find in the realm permissions table

Reply:
Actually just added this in this PR, https://github.com/cardstack/boxel/pull/1680/commits/dda441210f8490c936cd8417fd27a124b65c961f